justfile

```diff
@@ -1,4 +1,4 @@
-PROJECTS := "pytest-saas pytest-itde"
+PROJECTS := "pytest-backend pytest-saas pytest-itde"
```

Comment:
Looking at the other pytest plugins I have the feeling that "backend" is a bit too generic. 
Comparing the pypi package name in file `pyproject.toml` I think I would prefer `pytest-exasol-connection`.
What do you think?